### DISCUSSION TITLE
Help with repositories

### DISCUSSION THREAD

**Discussion by newbiethrough** [community/community#69568] [New to GitHub]
Hi, Beginner here 
Just wanted to ask; how can I share a html website to code with my friends,someone told me to use a repository on github or liveshare or something and alas I am here
Thanks in advance

**Comment by AnkurBansal18** [comment]
Hey @newbiethrough ,

Here are 2 of the possible ways you can achieve your req.

**Option 1: Using GitHub**

_Create a GitHub Account:_ In your case, this step has already been completed.

_Create a New Repository:_

Click on the "+" sign in the top right corner of your GitHub profile page and select "New repository."
Give your repository a name and choose your desired settings (public or private).
Initialize the repository with a README file.
Upload Your HTML Files: (just drag and drop from your system)

Clone the GitHub repository to your local machine using a Git client or GitHub Desktop.
Copy your HTML files, along with any CSS, JavaScript, or other assets (if any), into the local repository folder.
Commit and push the changes to GitHub using Git commands or the GitHub Desktop application.

_Share the GitHub Repository:_

Once your code is on GitHub, you can share the repository URL with your friends, and they can access your website's code. Make the repository public so that they can create pull requests and code along.


**Option 2: Using Visual Studio Code's Live Share**

Install Visual Studio Code: If you don't have Visual Studio Code (VS Code) installed, you can download it from [code.visualstudio.com]
[https://code.visualstudio.com/].

_Open Your Project in VS Code:_

Open VS Code.
Use the "Open Folder" option to open your website project.

_Install the Live Share Extension:_

In VS Code, go to the Extensions view by clicking the square icon on the sidebar or pressing Ctrl+Shift+X.
Search for "Live Share" and install the "Live Share" extension by Microsoft.

_Start a Live Share Session:_

Click the "Live Share" button in the status bar at the bottom of VS Code.
Share the Live Share link with your friends.

_Collaborate in Real-Time:_

Your friends can click the link you shared to join the Live Share session.
They will be able to see and edit your code in real time.

P.S. - If you didn't understand the GitHub option, please check out any video on YouTube and you will be good to go.
Using either GitHub or Live Share, you can easily share your HTML website project with your friends, allowing them to collaborate with you or simply view your code and website. Choose the option that best suits your needs, whether it's for collaborative coding or sharing your project for review.


Hope it helps :)
